### ISSUE TITLE
SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure false positive

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#1089]
```
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       8.1.0           The Doctrine Coding Standa...
slevomat/coding-standard                       6.x-dev ee75ad8 Slevomat Coding Standard f...
squizlabs/php_codesniffer                      3.5.6           PHP_CodeSniffer tokenizes ...
```
Consider the following code example:

```php
<?php

switch (true) {
    default:
    case 1:
        echo 2;
}
```
Try checking (looks like a false positive):
```
$ phpcs \
    -s \
    --standard=Doctrine \
    --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing,PSR2.ControlStructures.SwitchDeclaration \
    test.php

FILE: /home/morozov/Projects/phpcs-playground/test.php
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 4 | ERROR | [x] Expected 1 lines after "default", found 0.
   |       |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------

Time: 63ms; Memory: 10MB
```
Try fo fix:
```
$ phpcbf \
    --standard=Doctrine \
    --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing,PSR2.ControlStructures.SwitchDeclaration \
    test.php

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/phpcs-playground/test.php      FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 111ms; Memory: 10MB
```

**Comment by kukulich** [comment]
I'll modify it but it's BC break so it will be part of 7.0.0.

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/3c9b4729ddbb7d2a6a7cd4fca224534f633042cd
